packages/affiliates/libs/affiliates/utils.js

```diff
@@ -182,6 +181,33 @@ async function createGithubIssue({ auth, owner = "UMAprotocol", repo = "protocol
   });
 }
 
+async function requestGithubReviewers({ auth, owner, repo, pull_number, reviewers, ...rest }) {
+  assert(auth, "requires github auth credentials");
+  const octokit = new Octokit({
+    auth
+  });
+  return octokit.pulls.requestReviewers({
+    owner,
+    repo,
+    pull_number,
+    reviewers,
```

Comment:
Will this automatically request a couple of random reviewers from the UMA team?

Reply:
its not wired in to do this, but maybe eventually